pages/hpc/jobs.md

```diff
@@ -1,169 +1,257 @@
 
 ##### Example Job Scripts
 
+Below are sample script which can be sbatch'd to Slurm to run on a compute node.
+
+Below is explanation of of the SBATCH options used in our samples. These are Slurm directives and should be understood before submitting a job.
+
 ```
-Job Script 1 - Debug
+#!/bin/sh
+#
+# 	How long the job will run once it begins. If the job runs longer than what is
+# 	defined here, it will be cancelled by Slurm. 
+# 	If you make the expected time too long, it will
+# 	take longer for the job to start. 
+# 	Format:	dd:hh:mm:ss 
+
+#SBATCH --time=00:01:00
+
+#	Define how many nodes you need. We ask for 1 node
+
+#SBATCH --nodes=1
+
+# 	You can define the number of tasks with --ntasks-per-node
+
+#SBATCH --ntasks-per-node=1
+
+# 	How much memory do you need.
+# 	This will define memory this job requires. 
+
+#SBATCH --mem=10000
+
+# 	Give your job a name, so you can recognize it in the queue
+
+#SBATCH --job-name="example-debug-job"
+
+# 	Tell slurm the name of the file to write to
+
+#SBATCH --output=example-debug-job.out
+
+# 	Tell slurm where to send emails about this job
+
+#SBATCH --mail-user=myemailaddress@institution.edu
+
+# 	Tell slurm the types of emails to send.
+# 	Options: NONE, BEGIN, END, FAIL, ALL 
+
+#SBATCH --mail-type=end
+
+# 	Tell Slurm which cluster,partition and qos schedule this job.
+
+#SBATCH --partition=debug
+#SBATCH --qos=debug
+#SBATCH --cluster=ub-hpc
+
 ```
 
+
+To submit to the **debug partition on the ub-hpc cluster**, the slurm script would look like:
+
 ```
-Job Script 2 - general-compute
+#!/bin/sh
+#
+#SBATCH --time=00:01:00
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --mem=10000
+#SBATCH --job-name="example-debug-job"
+#SBATCH --output=example-debug-job.out
+#SBATCH --mail-user=myemailaddress@institution.edu
+#SBATCH --mail-type=end
+#SBATCH --partition=debug
+#SBATCH --qos=debug
+#SBATCH --cluster=ub-hpc
+
+#Let's start some work
+echo "Hello world from debug node: "`/usr/bin/uname -n`
+#Let's finish some work
 ```
 
+To submit to the **general-compute partition on the ub-hpc cluster**, the slurm script would look like:
+
 ```
-Job Script 3 - faculty partition
+#!/bin/sh
+#
+#SBATCH --time=00:01:00
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --mem=10000
+#SBATCH --job-name="example-general-compute-job"
+#SBATCH --output=example-general-compute-job.out
+#SBATCH --mail-user=myemailaddress@institution.edu
+#SBATCH --mail-type=end
+#SBATCH --partition=general-compute
+#SBATCH --qos=general-compute
+#SBATCH --cluster=ub-hpc
+
+#Let's start some work
+echo "Hello world from general-compute node: "`/usr/bin/uname -n`
+#Let's finish some work
+
 ```
 
+To submit to the privately owned **ezurek partition on the faculty cluster**, the slurm script would look like:
```

Comment:
I would recommend swapping the faculty cluster example with the scavenger one and remove the specific ezurek partition name and replace with something generic.